main/utils/launch.ts

```diff
@@ -0,0 +1,25 @@
+import Electron from 'electron';
+import fs from 'fs';
+import path from 'path';
+
+export const isFirstAppLaunch = (): boolean => {
+  const checkFile = path.join(
+    Electron.app.getPath('userData'),
+    '.app-launched'
```

Comment:
Has this been renamed from the electron-utils string? If it's been renamed, it will think everyone has opened the app for the first time regardless of if they have or not. We should maintain the old string.